libunwind/test/configs/llvm-libunwind-shared.cfg.in

```diff
@@ -10,8 +10,10 @@ link_flags = []
 if @LIBUNWIND_ENABLE_CET@:
     compile_flags.append('-fcf-protection=full')
 
-if '@CMAKE_SYSTEM_NAME@' == 'Linux':
-    link_flags.append('-Wl,--export-dynamic')
+# Add -Wl,--export-dynamic if supported by the linker (this CMake variable will
+# be empty for Apple platforms).
```

Comment:
for non-ELF platforms.

AIX linker doesn't have this option.

Reply:
I don't really like the idea of the comment in the round brackets. I think it would be better to just say something like.. ```On ELF platforms, add -Wl,--export-dynamic if supported by the linker.```